src/libraries/System.Net.Http.WinHttpHandler/ref/System.Net.Http.WinHttpHandler.cs

```diff
@@ -44,8 +44,11 @@ public partial class WinHttpHandler : System.Net.Http.HttpMessageHandler
         public System.Func<System.Net.Http.HttpRequestMessage, System.Security.Cryptography.X509Certificates.X509Certificate2, System.Security.Cryptography.X509Certificates.X509Chain, System.Net.Security.SslPolicyErrors, bool>? ServerCertificateValidationCallback { get { throw null; } set { } }
         public System.Net.ICredentials? ServerCredentials { get { throw null; } set { } }
         public System.Security.Authentication.SslProtocols SslProtocols { get { throw null; } set { } }
+        [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows10.0.2004")]
```

Comment:
Should not it be 10.0.19041 (ie. the real version number, not the marketing one)?

Reply:
AFAIK the `$"{year:2}{month:2}"` thing is the version, `19041` is the build number. See also: https://github.com/dotnet/designs/blob/main/accepted/2020/platform-exclusion/platform-exclusion.md

Reply:
That sounds confusing as hell since the TFMs use the build numbers (https://blogs.windows.com/windowsdeveloper/2020/11/10/announcing-c-winrt-version-1-0-with-the-net-5-ga-release/).

Reply:
Seems like a valid concern. @terrajobst any comments on this?

Reply:
The official documentation also uses the build numbers (https://docs.microsoft.com/en-us/dotnet/standard/analyzers/platform-compat-analyzer)

Reply:
Ok, I'm pretty sure now we should use Build here, and there is a bug in the design doc. Opened dotnet/designs#170 to clarify this. Thanks a lot for spotting this!